File: src/main/java/ru/dragonestia/apocalypse/item/CopperIngotItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class CopperIngotItem extends Item implements AdvItem {

    public CopperIngotItem(Integer meta, int count) {
        super(ApocalypseID.COPPER_INGOT, meta, count, "Copper ingot");
    }

    @Override
    public String getRuName() {
        return "Медный слиток";
    }

    @Override
    public String getIcon() {
        return "textures/items/ingot/copper";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/CopperIngotItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class CopperIngotItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.COPPER_INGOT, 0, count, "Copper ingot"), AdvItem {

    override fun getRuName(): String = "Медный слиток"

    override fun getIcon(): String = "textures/items/ingot/copper"

}